src/js/view/controls/rewind-display-icon.js

```diff
@@ -0,0 +1,35 @@
+define([
+    'utils/helpers',
+    'utils/ui',
+    'templates/display-icon.html'
+], function(utils, UI, Template) {
+
+    return class RewindDisplayIcon {
+
```

Comment:
I like having a space above the constructor. It makes the class and constructor more readable.